packages/api/test/__fixtures__/sdk/simple/index.ts

```diff
@@ -79,18 +75,22 @@ export default class SDK {
    * Multiple status values can be provided with comma separated strings
    *
    * @summary Finds Pets by status
-   * @param metadata Object containing all path, query, header, and cookie parameters to supply.
```

Comment:
I've changed these parameter docblocks to only be present if the parameter doesn't have a type on it so we can avoid a lot of this duplicate boilerplate everywhere.